tpl/tplimpl/template_embedded.go

```diff
@@ -84,7 +84,7 @@ func (t *GoHTMLTemplate) EmbedTemplates() {
       <pubDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</pubDate>
       {{ with .Site.Author.email }}<author>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</author>{{end}}
       <guid>{{ .Permalink }}</guid>
-      <description>{{ .Content | html }}</description>
+      <description>{{ .Summary | html }}</description>
```

Comment:
@rdwatters - I guess I'm function under the assumption MailChimp has picked the broadest compatibility option since they are integrated to by many services?  Not sure if you had a chance to actually look at their page?

Thanks for all the help.